src/components/VideoProgress.js

```diff
@@ -0,0 +1,17 @@
+import React, { useState, useEffect } from "react";
+import SingleVideoPage from "./SingleVideoPage";
+import Loading from "./Loading";
+
+export default function VideoProgress() {
+  const [isLoading, setIsLoading] = useState(true);
+
+  return (
+    <div>
+      {isLoading ? (
+        <Loading setIsLoading={setIsLoading} />
+      ) : (
+        <SingleVideoPage />
+      )}
+    </div>
+  );
+}
```

Comment:
Without having the DS team pass us messages, we really wouldn't be able to set progress, so I think we should remove this file